Error cycling fails on torsiondrives

Currently error cycling jobs in GH actions are failing. This corresponds to the first torsiondrive dataset we've submitted since QCF 0.50. I think something's borked with the error collection/traceback printing. The error message is as follows:

```

Run python ./management/lifecycle.py --states "Error Cycling" --reset-errors --set-priority --set-computetag
Traceback (most recent call last):
Found 17 with the 'tracking' label
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 987, in <module>
    main()
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 978, in main
Processing PR #343
Processing dataset 'submissions/2024-01-30-xtalpi-shared-fragments-torsiondrive-v1.0/dataset.json.bz2'
    submission.execute_state(board=board,
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 160, in execute_state
    return self.execute_errorcycle(pr_card, pr_state,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 249, in execute_errorcycle
    results.append(ds.execute_errorcycle(reset_errors=reset_errors,
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 470, in execute_errorcycle
    complete = self._errorcycle_torsiondrive(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 576, in _errorcycle_torsiondrive
    error_counts = self.count_unique_error_messages(errors, pretty_print=True)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/runner/work/qca-dataset-submission/qca-dataset-submission/./management/lifecycle.py", line 514, in count_unique_error_messages
    for id, error in errors_in:
        ^^^^^^^^^
TypeError: cannot unpack non-iterable int object
Error: Process completed with exit code 1.
```

--- a/management/lifecycle.py
+++ b/management/lifecycle.py
@@ -511,7 +511,7 @@ class SubmittableBase:
     def count_unique_error_messages(errors_in, pretty_print=False):
         errors = defaultdict(set)
     
-        for id, error in errors_in:
+        for id, error in errors_in.items():
             errors["\n".join([error[i] for i in ['error_type', 'error_message']])].add(id)
     
         errors = dict(errors)

